internal/kafka/internal/workers/cluster_mgrs/deprovisioning_clusters_mgr.go

```diff
@@ -78,7 +78,10 @@ func (m *DeprovisioningClustersManager) processDeprovisioningClusters() error {
 				errList.AddErrors(errors.Wrapf(err, "failed to reconcile deprovisioning cluster %s", cluster.ClusterID))
 			}
 		} else {
-			glog.V(10).Infof("skipping deprovisioning of %s cluster with ClusterID = %s", api.EnterpriseDataPlaneClusterType.String(), cluster.ClusterID)
+			glog.V(10).Infof("deregistering enterprise cluster ClusterID = %s", cluster.ClusterID)
```

Comment:
Side comment:
I keep seeing an increasing number of conditional blocks in the code that do "if it is enterprise do this, otherwise do that". This makes me think we might to abstract this kind of logic at another level.

Reply:
+1000